pkg/controller/core/resource_controller.go

```diff
@@ -127,7 +151,7 @@ func (r *Reconciler) _reconcile(claim corev1alpha1.ResourceClaim) (reconcile.Res
 	} else if handler == nil {
 		// handler is not found - log this but don't fail, let an external provisioner handle it
 		log.Info("handler for claim is unknown, ignoring reconcile to allow external provisioners to handle it", "claim", claim.GetName())
-		return Result, nil
+		return RequeueIfDirty, nil
```

Comment:
Good example: is the object dirty in this case, or we just piggy-backing on the "constant" that matches our need? Seems like the latter.

Reply:
I think you misunderstand what `RequeueIfDirty` means.

Reply:
Yes, you right, there is a bit of misunderstanding on my part. I left another comment above.